RunStuff Template.m

```diff
@@ -124,7 +124,7 @@
 "Yes"-> 0.,
 "mids"-> {-1,1},
 "freqs"->{0,2},
-"Endensity"-> {{{{ (1+a)/(munits/h),0.},{0.,0.}}},{{{0.,0.},{ (1-a)/(munits/h),0.}}}},
+"Endensity"-> {{{{0., 0.},{(1+a)/(munits/h),0.}}},{{{(1-a)/(munits/h),0.},{0.,0.}}}},

```

Comment:
Why does the first matrix have an entry in the off-diagonal element?

Reply:
The way it's indexed is data["Endnesity"][[species,theta bin (from left to right), phi bin]].  Since there's a sum over the phi bins, it doesn't actually matter which position it's in, so long as it's on the second row.